core/src/main/java/io/aiven/klaw/service/UtilControllerService.java

```diff
@@ -645,4 +642,12 @@ public void resetCache(String tenantName, String entityType, String operationTyp
       log.error("Error from resetCache ", e);
     }
   }
+
+  public Map<String, SSODetails> getLoginurl() {
+    return loginurl;
+  }
+
+  public void setLoginurl(Map<String, SSODetails> loginurl) {
```

Comment:
```suggestion
  public void setLoginurl(Map<String, SSODetails> loginUrl) {
```

Reply:
This just depends on if we update the application.properties